index.html

```diff
@@ -0,0 +1,292 @@
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="divider">
+          <hr class="solid" />
+        </div>
+        <div id="abilities">
+          <ul class="tech languages">
+            <li>Language</li>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+
+          <ul class="tech frameworks">
+            <li>Frameworks</li>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+
+          <ul class="tech skills">
+            <li>Skills</li>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+
+      <footer class="supporting-item">
+        <div class="element-left">
+          Work
+          <a href="#">
+            <img class="down-arrow" src="Down-arrow.png" alt="Down arrow" />
+          </a>
+        </div>
+        <div class="icons social-media">
+          <ul class="social-tags">
+            <li>
+              <img src="GitHub.png" alt="GitHub icon" />
+            </li>
+
+            <li>
+              <img src="LinkedIn.png" alt="LinkedIn icon" />
+            </li>
+
+            <li>
+              <img src="Angelist.png" alt="Angelist icon" />
+            </li>
+
+            <li>
+              <img src="Twitter.png" alt="Twitter icon" />
+            </li>
+
+            <li>
+              <img src="Medium.png" alt="Medium icon" />
+            </li>
+          </ul>
+          <div class="element-right">
+            <a href="#">
+              <img class="down-arrow" src="Down-arrow.png" alt="Down arrow" />
+            </a>
+            Work
+          </div>
+        </div>
+      </footer>
+    </section>
+    <!-- works list -->
+    <section id="recent-works">
+      <header class="works-header">
+        <h3>My recent work</h3>
+      </header>
+      <div class="divider-h3">
+        <hr class="border-h3" />
+      </div>
+
+      <div class="works-list">
+        <div class="featured-img">
+          <img src="ImgPlaceholder.png" alt="Image placeholder" />
+        </div>
+        <div class="right-block">
+          <h4>Multi-Post Stories</h4>
+          <p class="work-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="button-works">See Project</button>
+        </div>
+      </div>
```

Comment:
- [ ] There is too much spacing with `li` tag and it is separated with a small line as shown in the figure kindly implement it.
- [ ] Your are having black border in the see project buttons kindly implement it.
- [ ] Kinldy implement it in all the section
### your design
![his](https://user-images.githubusercontent.com/81727988/171835023-5e4244ae-97c9-4a96-a14b-2f288bbb775c.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171835044-3e4f1d25-36c4-4b23-a6da-67c2e323e61f.png)